planetarium/NineChronicles
# Fix Sword Skill's power

Fix Ring Recipe's stats

### Description

1. Fix Sword Skill's power
2. Fix Ring Recipe's stats
3. Fix Some recipe's materials

### How to test

1. Check Tables
2. Check Recipes in the craft menu

### Related Links

### Required Reviewers

@Namyujeong 

### Screenshot
![K-012](https://user-images.githubusercontent.com/60082897/134535353-f948a13e-9540-4736-8375-12b877287900.png)
![K-013](https://user-images.githubusercontent.com/60082897/134535368-a9f2fed1-632c-49aa-aef7-f3b0391abe43.png)
